src/EncodingTypes.cpp

```diff
@@ -2,5 +2,6 @@
 
 ENUMERATION_NAMING_BEGIN(EncodingTypes)
 {Thrift, "Thrift"},
-{ByteStream, "Byte Stream"}
-ENUMERATION_NAMING_END(EncodingTypes)
\ No newline at end of file
+{ByteStream, "Byte Stream"},
```

Comment:
There is no Bytestream serialization, it needs to be removed, only Thrift and Json are supported.